remind-bindings.el

```diff
@@ -23,22 +23,21 @@
 (require 'popwin)
 (require 'map)
 (require 'paren)
-(require 'org)
 
 (defgroup remind-bindings nil
   "Group for remembering bindings."
-  :group 'emacs)
+  :group 'convenience)
```

Comment:
nice, never knew about this group

Reply:
Ordinary packages do not define groups under Emacs categories.
Subscribing to or a subgroup is implied consent.
However, there are some packages that do not follow the rules...